src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
```

Comment:
Consider using a variable for the width and height of the product image to maintain consistency with other dimensions defined in variables.scss and to allow for easier maintenance and reusability.